### ISSUE TITLE
couldn't find or load Qt platform plugin "windows" in "".

### ISSUE THREAD

**Issue by ozdeger** [matplotlib/matplotlib#12702]
I am new to coding (also to github) and trying to learn CNN by research
i followed a full tutorial to create a MNIST handwriting guesser (ai)
then the tutorial i was learning from installed matplotlib to draw a Loss Graph
aaand this error started popping when any of matplotlib command were run (import matplotlib works)

i installed matplotlib from pycharm's settings...

![error](https://user-images.githubusercontent.com/44675543/47867670-3d5c3c80-de13-11e8-939c-bf7d0d161179.png)

i did check out other posts on this topic and did do the thing about ASPCll but didnt change anything
my user profile file name was Gökay changed in to Gokay still getting the same error...
```
My Code:

import tensorflow as tf
import matplotlib.pyplot as plt


from tensorflow.examples.tutorials.mnist import input_data
mnist = input_data.read_data_sets("data/MNIST/", one_hot=True)

x = tf.placeholder(tf.float32, [None, 784])
y_true = tf.placeholder(tf.float32, [None, 10])

layer_1 = 128
layer_2 = 64
layer_3 = 32
layer_out = 10


w1 = tf.Variable(tf.truncated_normal([784,layer_1],stddev=0.1))
b1 = tf.Variable(tf.constant(0.1,shape=[layer_1]))

w2 = tf.Variable(tf.truncated_normal([layer_1,layer_2],stddev=0.1))
b2 = tf.Variable(tf.constant(0.1,shape=[layer_2]))

w3 = tf.Variable(tf.truncated_normal([layer_2,layer_3],stddev=0.1))
b3 = tf.Variable(tf.constant(0.1,shape=[layer_3]))

w4 = tf.Variable(tf.truncated_normal([layer_3,layer_out],stddev=0.1))
b4 = tf.Variable(tf.constant(0.1,shape=[layer_out]))

y1 = tf.nn.relu(tf.matmul(x,w1) + b1)
y2 = tf.nn.relu(tf.matmul(y1,w2) + b2)
y3 = tf.nn.relu(tf.matmul(y2,w3) + b3)

logits = tf.matmul(y3, w4) + b4
y4 = tf.nn.softmax(logits)

xent = tf.nn.softmax_cross_entropy_with_logits(logits=logits,labels=y_true)
loss = tf.reduce_mean(xent)
correct_prediction = tf.equal(tf.argmax(y4, 1), tf.argmax(y_true, 1))
accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))

optimize= tf.train.AdamOptimizer(0.001).minimize(loss)

sess = tf.Session()
sess.run(tf.global_variables_initializer())

batch_size=128

loss_graph = []

def training_step(iterations):
    for i in range(iterations):
        x_batch, y_batch = mnist.train.next_batch(batch_size)
        feed_dict_train = {x: x_batch, y_true: y_batch}
        [_,train_loss] = sess.run([optimize,loss], feed_dict= feed_dict_train)

        loss_graph.append(train_loss)

        if i % 100 ==0:
            train_acc = sess.run(accuracy, feed_dict= feed_dict_train)
            print('iterations:',i,'Training Accuracy:', train_acc,'Training Loss:',train_loss)

def test_accuracy():
    feed_dict_test = {x: mnist.test.images, y_true: mnist.test.labels}
    acc = sess.run(accuracy, feed_dict=feed_dict_test)
    print("testing accuracy:", acc)

training_step(1000)
test_accuracy()

plt.plot(loss_graph,'k-')
plt.title('Loss Graph')
plt.xlabel('Iteration')
plt.ylabel('Loss')
```

**Matplotlib version**
  * Operating system: win10 home
  * Matplotlib version:  i have tried 3.0.0/2.2.0/2.0.0
  * Python version: 3.5
  * Other libraries: 
a lot.... i can paste them if needed
(i have tensorflow-gpu)

**Comment by tacaswell** [comment]
Can you reproduce this issue without tensorflow?

**Comment by ozdeger** [comment]
sure shouldnt be that hard
![error3](https://user-images.githubusercontent.com/44675543/47877126-2ece4f00-de2c-11e8-91ee-1ac448f1ee35.png)
there it is again

**Comment by tacaswell** [comment]
@ozdeger I am glad it is working for you now!

For future readers, my suspicion is that this was a Qt installation / configuration issue (as is suggested by the error box!)
